src/components/Search.component.jsx

```diff
@@ -0,0 +1,169 @@
+import React from "react";
+
+export const Search = (props) => {
```

Comment:
the name may be misleading if we want to use it not only for search, it would be better to change it to simply 'Input'

Reply:
ok on this too lol